scripts/generate-content-for-preview-antora-playbook.js

```diff
@@ -13,6 +13,7 @@ const repoUrls = new Map([
     ['bcd', 'https://github.com/bonitasoft/bonita-continuous-delivery-doc.git'],
     ['bonita', 'https://github.com/bonitasoft/bonita-doc.git'],
     ['cloud', 'https://github.com/bonitasoft/bonita-cloud-doc.git'],
+    ['test-toolkit', 'https://github.com/bonitasoft/bonita-test-toolkit-doc.git'],
     ['labs', 'https://github.com/bonitasoft/bonita-labs-doc.git'],
```

Comment:
Let's keep the alphabetical order here
```suggestion
    ['labs', 'https://github.com/bonitasoft/bonita-labs-doc.git'],
    ['test-toolkit', 'https://github.com/bonitasoft/bonita-test-toolkit-doc.git'],
```